internals/secrethub/service_aws_init.go

```diff
@@ -85,13 +86,19 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 
 	fmt.Fprintf(cmd.io.Stdout(), "Detected access to AWS account %s.", accountID)
 
-	region := sess.Config.Region
-	if isSet(region) {
-		fmt.Fprintf(cmd.io.Stdout(), "Using region %s.", *region)
+	if !isSet(cfg.Region) && cmd.kmsKeyID != "" {
```

Comment:
Consider adding a comment here that explains what it does. It's a bit confusing to read. There are multiple `if isSet` and `if !isSet` statements here and there is an additional condition `if err != nil` inside here that is intentionally ignored.